prov/efa/src/efa_mr.c

```diff
@@ -152,6 +152,7 @@ static int efa_mr_cache_regattr(struct fid *fid, const struct fi_mr_attr *attr,
 	struct ofi_mr_entry *entry;
 	int ret;
 	static const int EFA_MR_CACHE_FLUSH_CHECK = 512;
+	static int call_counter = 0;
```

Comment:
I'd make this unsigned since it could wrap around. Also, perhaps change the name to `reg_counter`?

Reply:
Done. I change the type to "size_t" which is unsigned and 64 bits.